### PR TITLE
OCPQE-26461 Add test job registry config for 4.15

### PR DESCRIPTION
**Pull request by jhuttana**
Add test job registry config for 4.15

### CODE REVIEW

**Comment by jhuttana** [comment]
@rioliu-rh I am not sure whether [_releases/ocp-4.15-test-jobs-multi.json](url) file also needs some modification for nightly or not. If yes then what it should be?

**Comment by rioliu-rh** [comment]
> @rioliu-rh I am not sure whether [_releases/ocp-4.15-test-jobs-multi.json](url) file also needs some modification for nightly or not. If yes then what it should be?

currently we don't have controller job to run test for multi release stream.

**Comment by rioliu-rh** [comment]
this PR depends on https://github.com/openshift/release/pull/58145

File: _releases/ocp-4.15-test-jobs-amd64.json [hunk 1]
```diff
@@ -2,7 +2,37 @@
   "nightly": [
     {
       "prowJob": "periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-aws-ipi-fips-f1"
```
[the remark this answers is not in the document]
Comment (jhuttana) [reply _releases/ocp-4.15-test-jobs-amd64.json]: Done

**Pull request merged**